setup.py

```diff
@@ -66,8 +66,9 @@
     "datasets",
 ]
 EXTRAS = {
-    "test": ["parameterized", "pytest", "pytest-xdist", "accelerate"],
-    "dev": ["parameterized", "pytest", "pytest-xdist", "black", "isort", "flake8>=3.8.3"],
+    "test": ["parameterized", "pytest", "pytest-xdist", "accelerate", "peft"],
+    "peft": ["parameterized", "pytest", "pytest-xdist", "accelerate", "peft"],
```

Comment:
I think the test stuff is not needed and accelerate is already a core dependency, no?
```suggestion
    "peft": ["peft"],
```